collateral/engine.go

```diff
@@ -108,19 +113,15 @@ func New(log *logging.Logger, conf Config, broker Broker, now time.Time) *Engine
 		accs:          make(map[string]*types.Account, initialAccountSize),
 		partiesAccs:   map[string]map[string]*types.Account{},
 		hashableAccs:  []*types.Account{},
+		timeService:   ts,
 		broker:        broker,
-		currentTime:   now.UnixNano(),
 		idbuf:         make([]byte, 256),
 		enabledAssets: map[string]types.Asset{},
 		state:         newAccState(),
 	}
 }
 
-// OnChainTimeUpdate is used to be specified as a callback in over services
-// in order to be called when the chain time is updated (basically EndBlock).
-func (e *Engine) OnChainTimeUpdate(_ context.Context, t time.Time) {
-	e.currentTime = t.UnixNano()
-}
+func (e *Engine) OnChainTimeUpdate(context.Context, time.Time) {}
```

Comment:
hmm, so why...?

Reply:
Yes, this can be removed - it is from Collateral interface;
I am going to remove it from the interface, so it will not be required for objects that implement it - currently it is only collateral.
Other engines have other definitions of it.

Reply:
It has been removed here https://github.com/vegaprotocol/vega/pull/5475/commits/e84526f90d9ea2a19a8fb841c85e5f07f0bb76db
Thanks!